subsys/bluetooth/mesh/sar_cfg_srv.c

```diff
@@ -26,6 +26,22 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_mesh_sar_cfg_srv);
 
+static int sar_rx_store(struct bt_mesh_model *model, bool delete)
```

Comment:
I would guess that this model is more like the cfg models where the data should not be altered very often. Besides, the CONFIG_BT_MESH_MODEL_SRV_STORE_TIMEOUT option resides in NCS model Kconfig options, so that would require a new simmilar option in Zephyr